server/src/models/User.js

```diff
@@ -32,18 +32,45 @@ class User extends uniqueFunc(Model) {
         email: { type: "string" },
         cryptedPassword: { type: "string" },
       },
-    };
+    }
   }
 
   $formatJson(json) {
-    const serializedJson = super.$formatJson(json);
+    const serializedJson = super.$formatJson(json)
 
     if (serializedJson.cryptedPassword) {
-      delete serializedJson.cryptedPassword;
+      delete serializedJson.cryptedPassword
     }
 
-    return serializedJson;
+    return serializedJson
+  }
+  static get relationMappings() {
+    const { Review, Trail } = require("./index")
+
+    return {
+      reviews: {
+        relation: Model.HasManyRelation,
+        modelClass: Review,
+        join: {
+          from: "users.id",
+          to: "reviews.userId",
+        },
+      },
+
+      trails: {
```

Comment:
What is your join table/ association between a Trail and User for needing a many-to-many?

A User has many Trails
A Trail belongs to a User

Reply:
When querying the users - the instance of a user via the relational mapping will allow us to see the trails reviewed by the user.

Reply:
See my above comment for how we should define this association between a User and Trail